.github/workflows/w1-workflow-build-matrix.yml

```diff
@@ -152,6 +152,7 @@ jobs:
               python3 -m pip install --upgrade pip
               pip3 uninstall -y PKNSETools
               pip3 uninstall -y PKDevTools
+              cd ../..
```

Comment:
but we are already at the root directory.
```
cd /home/runner/work/PKScreener/PKScreener/
```
@Redbro1

Reply:
See attached build logs. Rejecting the PR.
[2_Build Packages (ubuntu-20.04, Linux, pyinstaller --onefile --icon=screenshots_icon.ico pkscreener....txt](https://github.com/user-attachments/files/16929872/2_Build.Packages.ubuntu-20.04.Linux.pyinstaller.--onefile.--icon.screenshots_icon.ico.pkscreener.txt)